src/core/order_manager.py

```diff
@@ -0,0 +1,25 @@
+# order manager mechanism
+
+from queue import Queue
+
+
+class OrderManager:
+    def __init__(self):
+        self._order_storage = {
+            "order_placed": Queue(maxsize=0),
+            "in_process": Queue(maxsize=0),
+            "cancelled": Queue(maxsize=0),
+            "completed": Queue(maxsize=0),
+        }
+
+    def add_to_order_storage(self, order):
+        self._order_storage[order.order_status.value].put(order)
+
+    def get_from_order_storage(self, order_status):
```

Comment:
Could you rename to `get_queue_from_status`?

Reply:
done